package-dev/Plugins/Android/proguard-sentry-unity.pro

```diff
@@ -1,9 +1,5 @@
 # Unity: keep names on select sentry-java classes & their methods - we use string-based JNI lookup in our integration.
--keep class io.sentry.Sentry { *; }
--keep class io.sentry.SentryLevel { *; }
--keep class io.sentry.SentryOptions { *; }
--keep class io.sentry.Hub { *; }
--keep class io.sentry.Breadcrumb { *; }
--keep class io.sentry.Scope { *; }
--keep class io.sentry.ScopeCallback { *; }
--keep class io.sentry.protocol.** { *; }
+-keep class io.sentry.** { *; }
+-keep class io.sentry.android.** { *; }
```

Comment:
how is this not the same thing?

Reply:
Fair point. I've updated it and added only what we specifically need.